playbooks/preflight-check.yml

```diff
@@ -138,3 +138,18 @@
     - name: Duplicate flavors exist
       fail: msg="Duplicated flavor IDs are detected."
       when: flavor_check.stdout != ""
+
+# RLM-1371 detect if enough space for rootfs backup
+- name: Check space for rootfs backup on infras
+  hosts: "os-infra-hosts"
+  user: root
+  tasks:
+    - name: Indentify the space used in /
+      shell: |
+        df -h / | tail -n 1 | awk '{print $5}' | sed 's/%//'
+      register: rootfs_space_used
+
+    - name: Check space if enough
```

Comment:
Might be good to make the description a bit more verbose so operators know the threshold without having to dive into the code in the event of a failure:

"Check if there is at least 50% space available on Infra nodes"